linux_os/guide/system/software/system-tools/package_sudo_installed/rule.yml

```diff
@@ -16,9 +16,9 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@rhel6: 82967-1
-    cce@rhel7: 82966-3
-    cce@rhel8: 82965-5
```

Comment:
Please change this also in `rhel8/profiles/ospp.profile`.

Reply:
OK